public/script.js

```diff
@@ -5469,15 +5469,15 @@ async function promptItemize(itemizedPrompts, requestedMesId) {
     } else {
         diffPrevPrompt.style.display = 'none';
     }
-    popup.dlg.querySelector('#copyPromptToClipboard').addEventListener('click', function () {
+    popup.dlg.querySelector('#copyPromptToClipboard').addEventListener('pointerup', async function () {
```

Comment:
Why? Did you see the need to capture right click and middle mouse as well for the copy? that's a bit weird.
Or like, you could click somewhere, drag and end up on the copy button and it copies?

Not how I'd expect a copy button to work.

Reply:
Blergh. Weird browser workarounds again.
It's just weird on any "normal" browser now, because yeah. Right click is the most glaring one. Dunno if there is a way around it.
Could maybe check for the mouse button pressed int the event handler.